modules/ossm-kiali-accessing-console.adoc

```diff
@@ -0,0 +1,46 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+* service_mesh/v2x/ossm-troubleshooting-istio.adoc
+////
+
+[id="ossm-accessing-kiali-console_{context}"]
+= Accessing the Kiali console
+
+You can view your application's topology, health, and metrics in the Kiali console. If your service is experiencing problems, the Kiali console lets you view the data flow through your service. You can view insights about the mesh components at different levels, including abstract applications, services, and workloads. Kiali also provides an interactive graph view of your namespace in real time.
```

Comment:
This is good but I think we could go further here.  Kiali is not just for viewing, and not only to be used after you have some sort of problem.  I think we should start promoting it as The Service Mesh UI, or Console.   Up front it can be used to help generate and validate your Istio config.

Also, the directions below are for accessing Kiali from OS console, but perhaps we should add instructions for accessing directly?

Reply:
I'm with you, I'm not happy about how far into the docs we've buried the content for accessing Kiali.  I'm going to be working with the rest of the team on some new content strategy next year, and one of the things I plan to address is making it more clear that Kiali is the management console for Service Mesh.

When you say "Accessing Kiali directly" do you mean just going straight to the URL for the route?

Reply:
I guess so, I just wanted to ensure that users don't think they have to access Kiali via console.

Reply:
I added a line, "If you know the URL for the Kiali console, you can access it directly.  If you do not know the URL, use the following directions."